src/schema/v2/partner/__tests__/partner.test.js

```diff
@@ -156,6 +156,70 @@ describe("Partner type", () => {
     })
   })
 
+  describe("Partner loader selection", () => {
```

Comment:
Eh, you don't need a test for this - we typically don't when adding same-named authenticated loaders - but why not!

Reply:
yeah it does feel a little force feed-y doesn't it?
Maybe I can use them to help me sleep easy at night to see it working but not commit them 😄